src/pdf/DriverOverview/DriverOverviewPdfButton.tsx

```diff
@@ -43,7 +46,32 @@ type ParcelsForDeliveryResponse =
           error: { type: ParcelsForDeliveryErrorType; logId: string };
       };
 
-type ParcelsForDeliveryErrorType = "parcelFetchFailed" | "noMatchingClient";
+type ParcelsForDeliveryErrorType = "parcelFetchFailed" | "noMatchingClient" | "eventFetchFailed";
+
+type ParcelEventResponse =
+    | {
+          data: Schema["events"][];
+          error: null;
+      }
+    | {
+          data: null;
+          error: { type: ParcelsForDeliveryErrorType; logId: string };
+      };
+
+// Gets all "shipping labels downloaded" events for the given parcels
+const getParcelEvents = async (parcelIds: string[]): Promise<ParcelEventResponse> => {
+    const { data, error } = await supabase
+        .from("events")
+        .select("*")
+        .in("parcel_id", parcelIds)
+        .eq("new_parcel_status", "Shipping Labels Downloaded")
+        .order("timestamp", { ascending: false });
+    if (error) {
```

Comment:
can you add an empty line above this if block, and also one below it, for readability